allennlp/modules/seq2seq_encoders/bidirectional_transformer_encoder.py

```diff
@@ -0,0 +1,265 @@
+"""
+The BidirectionalTransformerEncoder from calypso
```

Comment:
I think we should link the annotated transformer tutorial (http://nlp.seas.harvard.edu/2018/04/03/attention.html) here as this implementation appears to be based on that. MIT license conveniently. https://github.com/harvardnlp/annotated-transformer/blob/master/LICENSE